web/themes/custom/usagov/scripts/callCenterWaitTimeLookup.js

```diff
@@ -0,0 +1,85 @@
+jQuery(document).ready(async function () {
+  "use strict";
+
+  var waittime = (function() {
+    if (jQuery("html[lang|='en']").length ||
+        jQuery("html[lang|='es']").length) {
+      var domain = window.location.origin.split('.')[0].split('//')[1];
+      var domainMap = {
+        'www': 'cms',
+        'cms': 'cms',
+        'beta': 'cms',
+        'beta-stage': 'cms-stage',
+        'cms-stage': 'cms-stage',
+        'beta-dev': 'cms-dev',
+        'cms-dev': 'cms-dev',
+        'localhost': 'localhost',
+      };
+      jQuery.ajax({
+        "url": "https://" + domainMap[domain] + ".usa.gov/wait-time",
+        "type": "GET",
+        "success": function (response) {
+          var json = jQuery.parseJSON(response);
+          var seconds = -1;
+          if (jQuery("html[lang|='en']").length) {
+            seconds = json.enEstimatedWaitTimeSeconds;
+          }
+          if (jQuery("html[lang|='es']").length) {
+            seconds = json.spEstimatedWaitTimeSeconds;
+          }
+
+          if (seconds >= 0) {
```

Comment:
I would also like the ccwaittimes script to include a timestamp in the file it writes to s3, and for this script to check it, so that if the former has stopped updating the file, the latter will detect that and not display an estimate. 

(I realize these comments are really for Jacob.)

Reply:
See ticket USAGOV-1945